frontend/js/src/common/brainzplayer/VolumeControlButton.tsx

```diff
@@ -1,20 +1,36 @@
 import * as React from "react";
-import { useBrainzPlayerDispatch } from "./BrainzPlayerContext";
+import {
+  useBrainzPlayerDispatch,
+  useBrainzPlayerContext,
+} from "./BrainzPlayerContext";
+
+const STORAGE_KEY = "brainzplayer-volume";
```

Comment:
To avoid any potential confusion in the future, I suggest a less generic name:
```suggestion
const BP_VOLUME_STORAGE_KEY = "brainzplayer-volume";
```